TF_Tooling_Web/examples/SnappingPlugin/Snapping_Example.html

```diff
@@ -0,0 +1,122 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Quick Start Example</title>
+    <script name="TouchFree_Tooling" type="text/javascript" src="../dist/TouchFree_Tooling.js"></script>
```

Comment:
When I build and run locally, this path isn't found. I needed to adjust to `../../dist/Touchfree_Tooling.js`